src/scripts/migrate.py

```diff
@@ -75,8 +75,7 @@ def slurp(
                 break
         if qualified_parents:
             next_mondo_id, mondo_term_ids = _get_next_available_mondo_id(next_mondo_id, max_id, mondo_term_ids)
-            # mondo_id = 'MONDO:' + ('0' * (len(str(next_mondo_id)) - 7)) + str(next_mondo_id)  # min->max needs no pad
-            mondo_id = 'MONDO:' + str(next_mondo_id)
+            mondo_id = 'MONDO:' + ('0' * (len(str(next_mondo_id)) - 7)) + str(next_mondo_id)  # min->max needs no pad
```

Comment:
The padding

Reply:
I  think it is more robust to use zfill:

https://stackoverflow.com/questions/39402795/how-to-pad-a-string-with-leading-zeros-in-python-3

Reply:
Ok; didn't even know that existed. This is such a simple programming problem that I didn't even think twice. I'll switch to use that, though.

Reply:
done